webportal/lambda/deployment/deploy_lambda.py

```diff
@@ -0,0 +1,9 @@
+from . import cli_parser, gateway_normalizer
+import subprocess
+
+
+def deploy_backend(command_line_arguments):
+    parsed_arguments = cli_parser.parse_arguments(command_line_arguments)
+    chalice_deploy_command = f'chalice deploy --stage {parsed_arguments.environment} --no-autogen-policy --profile sdc'
```

Comment:
lol I like that we've obscured the `stage` concept because chalice is too opinionated.

Reply:
Yeah, Chalice always wanted to use stages... but when this first got stood up we went with 3 separate environments. When we transition to ECS its possible we can re-examine this...